manifest.json

```diff
@@ -13,7 +13,11 @@
     }
   },
 
-  "permissions": ["telemetry"],
+  "permissions": [
+    "telemetry",
+    "storage",
+    "https://firefox.settings.services.mozilla.com/"
```

Comment:
Should we have something in the `README` about this? There is a devserver that we can use for testing for instance: https://remote-settings.readthedocs.io/en/latest/tutorial-dev-server.html so it'd be nice to mention :)

Reply:
That would be nice. I'm not sure I fully follow – let's sync up on this so I understand the details